test/hotspot/jtreg/runtime/signal/SigTestDriver.java

```diff
@@ -146,4 +170,27 @@ private static List<String> vmargs() {
     private static Path libjsig() {
         return Platform.jvmLibDir().resolve(Platform.buildSharedLibraryName("jsig"));
     }
+
+    /**
+     * Return true for all signals used by the JVM. This only covers the
+     * case where the JVM is started normally, and not with -Xrs.
```

Comment:
The VM also uses SIGUSR2 (`SR_signum`), SIGINT, and SIGQUIT. Though the latter two only if no `-Xrs` is given.

Reply:
The test is skipped for SIGUSR2 on Linux and MacOS, and Windows skips all signal testing. So I guess that leaves aix, which seems to use signals_poxis.cpp... so perhaps the SIGUSR2 test should be skipped for aix too?
But I'll add the signals for clarity.

Reply:
Just to add to this, "aix" is explicitly skipped in the TestSigsur2.java test. So this test is never run for SIGUSR2.

I changed the comment for the method to indicate better what it returns.

Reply:
Hmmm I had missed the skipping of SIGUSR2 - not sure why that is the case. But okay.